packages/viem/src/actions/sendSuperchainWETH.ts

```diff
@@ -0,0 +1,107 @@
+/** @module sendSuperchainWETH */
+import type {
+  Account,
+  Address,
+  Chain,
+  Client,
+  DeriveChain,
+  Transport,
+} from 'viem'
+
+import { contracts } from '@/contracts.js'
+import type { BaseWriteContractActionParameters } from '@/core/baseWriteAction.js'
+
+import type {
+  SendSupERC20ContractReturnType,
+  SendSupERC20ReturnType,
+} from './sendSupERC20.js'
+import {
+  estimateSendSupERC20Gas,
+  sendSupERC20,
+  simulateSendSupERC20,
+} from './sendSupERC20.js'
+
+/**
+ * @category Types
+ */
+export type SendSuperchainWETHParameters<
+  TChain extends Chain | undefined = Chain | undefined,
+  TAccount extends Account | undefined = Account | undefined,
+  TChainOverride extends Chain | undefined = Chain | undefined,
+  TDerivedChain extends Chain | undefined = DeriveChain<TChain, TChainOverride>,
+> = BaseWriteContractActionParameters<
+  TChain,
+  TAccount,
+  TChainOverride,
+  TDerivedChain
+> & {
+  /** Address to send tokens to. */
+  to: Address
+  /** Amount of tokens to send. */
+  amount: bigint
+  /** Chain ID of the destination chain. */
+  chainId: number
+}
+
+/**
+ * Sends SuperchainWETH to a target address on another chain. Used in the interop flow.
+ * @category L2 Wallet Actions
+ * @param client - L2 Wallet Client
+ * @param parameters - {@link SendSuperchainWETHParameters}
+ * @returns The sendSuperchainWETH transaction hash. {@link SendSupERC20ReturnType}
+ */
+export async function sendSuperchainWETH<
+  chain extends Chain | undefined,
+  account extends Account | undefined,
+  chainOverride extends Chain | undefined = undefined,
+>(
+  client: Client<Transport, chain, account>,
+  parameters: SendSuperchainWETHParameters<chain, account, chainOverride>,
+): Promise<SendSupERC20ReturnType> {
+  return sendSupERC20(client, {
```

Comment:
When we start to autogenerate all of this code this portion will change since they will all follow the same identical structure. Shouldn't cause any sort of breaking change